plaso/storage/redis/redis_store.py

```diff
@@ -0,0 +1,488 @@
+      return
+    container_key = self._GenerateRedisKey(container_type)
+    string_identifiers = [
+        identifier.CopyToString() for identifier in container_identifiers]
+
+    self._redis_client.hdel(container_key, *string_identifiers)
+    if container_type == self._CONTAINER_TYPE_EVENT:
+      event_index_name = self._GenerateRedisKey(self._EVENT_INDEX_NAME)
+      self._redis_client.zrem(event_index_name, *string_identifiers)
+
+  def IsFinalized(self):
+    """Checks if a store has been finalized.
+
+    Returns:
+      bool: True if the store has been finalized.
+    """
+    self._RaiseIfNotReadable()
+
+    finalized_key = self._GetFinalizationKey()
+    finalized_value = self._redis_client.hget(
+        finalized_key, self._task_identifier)
+
+    return finalized_value == self._FINALIZED_BYTES
+
+  def Finalize(self):
+    """Marks a store as finalized.
+
+    No further attribute containers will be written to a finalized store.
+    """
+    self._RaiseIfNotWritable()
+
+    finalized_key = self._GetFinalizationKey()
+    self._redis_client.hset(
+        finalized_key, self._task_identifier, self._FINALIZED_BYTES)
+
+  def GetSortedEvents(self, time_range=None):
+    """Retrieves the events in increasing chronological order.
+
+    Args:
+      time_range (Optional[TimeRange]): This argument is not supported by the
+          redis store.
+
+    Yields:
+      EventObject: event.
+
+    Raises:
+      RuntimeError: if a time_range argument is specified.
+    """
+    event_index_name = self._GenerateRedisKey(self._EVENT_INDEX_NAME)
+    if time_range:
+      raise RuntimeError('Not supported')
+
+    sorted_event_identifiers = self._redis_client.zscan_iter(event_index_name)
+    for event_identifier, _ in sorted_event_identifiers:
+      identifier_string = event_identifier.decode('utf-8')
+      event_identifier = identifiers.RedisKeyIdentifier(identifier_string)
+      yield self._GetAttributeContainerByIdentifier(
+          self._CONTAINER_TYPE_EVENT, event_identifier)
+
+  def GetSerializedAttributeContainers(
+      self, container_type, cursor, maximum_number_of_items):
+    """Fetches serialized attribute containers.
+
+    Args:
+      container_type (str): attribute container type.
+      cursor (int): Redis cursor.
+      maximum_number_of_items (int): maximum number of containers to
+          retrieve, where 0 represent no limit.
+
+    Returns:
+      tuple: containing:
+        int: Redis cursor.
+        list[bytes]: serialized attribute containers.
+    """
+    name = self._GenerateRedisKey(container_type)
+    # Redis treats None as meaning "no limit", not 0.
+    if maximum_number_of_items == 0:
+      maximum_number_of_items = None
+
+    cursor, items = self._redis_client.hscan(
+        name, cursor=cursor, count=maximum_number_of_items)
+    return cursor, items
+
+  # pylint: disable=arguments-differ
+  def Open(self, redis_client=None, url=None):
```

Comment:
try adding `, **unused_kwargs` and removing the pylint override

Reply:
Pylint still doesn't like that

Reply:
Which version of pylint are you using?

Reply:
I'm able to reproduce this with pylint 2.4.4, leave it as is

Reply:
I still would as to add unused_kwargs

Reply:
Done